convert/convert.go

```diff
@@ -119,28 +146,19 @@ func BedValuesToWig(inFile string, reference map[string]chromInfo.ChromInfo, Mis
 			log.Fatalf("Unrecognized method.")
 		}
 	}
+	//stable sort the wigSlice by Chrom
+	//so that each wig output will be the same as long as input is the same
+	//otherwise wig output will shift even if input is the same, due to the reference map
+	sort.SliceStable(wigSlice, func(i, j int) bool { return wigSlice[i].Chrom < wigSlice[j].Chrom })
```

Comment:
Same comment about not completely understanding why []wig.Wig needs to be in a certain order.